benchexec/tablegenerator/columns.py

```diff
@@ -18,12 +18,15 @@
 
 __all__ = ["Column", "ColumnType", "ColumnMeasureType"]
 
-# This sets the rounding mode for all Decimal operations in the process.
-# It is actually used only as default context for new contexts, but because we set this
-# at import time and before any threads are started, it should work according to its
-# documentation. We double check with the context of the current thread.
+# This sets the rounding mode for all Decimal operations in the process and explicitly for the current context
+# As asserting the rounding mode for the current context is almost a tautology, we also assert in the methods
```

Comment:
True, but for those who read it and do not know the history of this line, it would be quite confusing to state this and then still do the tautological assertion. I think that we can just get rid of the assert and this explanation then, but we would need to explain why checks in the methods are needed, and ideally a reference to #991.